web/src/test-data/index.ts

```diff
@@ -1 +1,5 @@
 export * from './factories/album-factory';
```

Comment:
We used to have these types of files in the server and it caused several problems. I'd rather avoid them.

Reply:
Alright I've removed this file